dbt/include/snowflake/macros/materializations/test.sql

```diff
@@ -0,0 +1,49 @@
+{%- materialization test, adapter='snowflake' -%}
```

Comment:
I'd hope that all of this could just be:
```
{%- materialization test, adapter='snowflake' -%}
    {% set original_query_tag = set_query_tag() %}
    {% set relations = materialization_test_default() %}
    {% do unset_query_tag(original_query_tag) %}
    {{ return(relations) }}
{% endmaterialization %}
```

Reply:
Right on! had that initially, then went to the full copy from dbt-core. Will shift back, test etc. Thanks @jtcohen6

Reply:
Hey @jtcohen6 with this set up, I can configure query tags to run on tests from the `dbt_project.yml`: 

```
tests:
  dev_project_snowflake:
    example:
      +query_tag: mw_dev_test
```

BUT, I can't set them from a `schema.yml`: 

```
  - name: my_second_dbt_model
    description: "A starter dbt model"
    columns:
      - name: id
        description: "The primary key for this table"
        tests:
          - unique:
              config:
                query_tag: mw_dev_test #nothing happens on snowflake with this setup
          - not_null
```

Should I be able to make this setting in both places?

Reply:
@matt-winkler Good catch. Unfortunately, this will have to come as a separate unit of work: https://github.com/dbt-labs/dbt-core/issues/5532